app/models/quiz_runner.rb

```diff
@@ -41,4 +42,14 @@ def complete_quiz
     save
   end
 
+  private
+
+  def user_pays_after_second_quiz
+    if QuizRunner.where(user_id: user.id).count >= 2
+      unless QuizOrder.exists?(user_id: user.id, quiz_id: quiz.id, status: 'paid')
```

Comment:
lets discuss this, will user be able to play any number of quizzes after playing once??